### PR TITLE
Detect anna@wire.com as bot

### PR DESCRIPTION
**Pull request by mikeger**
Currently Anna is having an email anna@wire.com, and it is not matching to our regex

### CODE REVIEW

File: Source/Model/User/ZMUser.m [hunk 1]
```diff
@@ -74,7 +74,7 @@
 static NSString *const ShowingUserRemovedKey = @"showingUserRemoved";
 static NSString *const UserClientsKey = @"clients";
 
-static NSString *const UserBotEmailRegex = @"^(welcome|anna)\\+(.*)@wire\\.com$";
+static NSString *const UserBotEmailRegex = @"^(welcome|anna)(.*)@wire\\.com$";
```
Comment (marcoconti83) [inline Source/Model/User/ZMUser.m]: If in one month we will hire a `annalisa`, she will be considered a bot :)
It think it's better to use:
`(\\+(.*))?`

**Comment by mikeger** [comment]
@marcoconti83 re-review please

**Comment by agentf90** [comment]
dafuq is this shit? my mistress is named anna and looks like this chick. you could imagine the scare i had when I installed wire app to talk to my wife and she shows up. can you identity bots as bot somehow? please?

**Comment by marcoconti83** [comment]
Hi @agentf90, thanks for the feedback. We are working on a new way to interact with bots and we will implement some changes in this area. Stay tuned!

**Pull request merged**